src/components/App/App.jsx

```diff
@@ -1,16 +1,26 @@
 import './App.css';
 import React from 'react';
 import PropTypes from 'prop-types';
+import Home from '../Home/Home.jsx'
+// import Navigation from '../Navigation/Navigation.jsx'
+import { Routes, Route, Link } from 'react-router-dom';
 
-function App() {
+export default function App() {
   return (
     <div className="App">
-      Content TBD
+      <header>
+        <Link to={'/'}><h1>Tavern Keeper</h1></Link>
```

Comment:
FYI Uncommenting this line 12 wasn't necessary to clear the warning, but totally fine to add. I probably would have done it with the Navigation ticket anyway. 😄